javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -227,15 +266,26 @@ object Character {
   }
 
   def isISOControl(c: scala.Char): scala.Boolean = isISOControl(c.toInt)
-  def isISOControl(codePoint: scala.Int): scala.Boolean = {
+
+  def isISOControl(codePoint: Int): scala.Boolean = {
     (0x00 <= codePoint && codePoint <= 0x1F) || (0x7F <= codePoint && codePoint <= 0x9F)
   }
 
-  def isDigit(c: scala.Char): scala.Boolean = c >= '0' && c <= '9'
-  //def isLetter(c: scala.Char): scala.Boolean
-  //def isLetterOrDigit(c: scala.Char): scala.Boolean
-  def isWhitespace(c: scala.Char): scala.Boolean = js.RegExp("^\\s$").test(c.toString)
-  //def isSpaceChar(c: scala.Char): scala.Boolean
+  @deprecated("Replaced by isWhitespace(char)", "")
+  def isSpace(c: scala.Char): scala.Boolean =
+    c == '\t' || c == '\n' || c == '\f' || c == '\r' || c == ' '
+
+  def isWhitespace(c: scala.Char): scala.Boolean =
+    js.RegExp("^\\s$").test(c.toString)
```

Comment:
The overload taking an `Int` is missing. It would also be a good opportunity to check that this regex-based implementation is really accurate. Maybe we should just use the same infrastructure as everything else anyway, at this point?